README.md

```diff
@@ -274,6 +277,44 @@ client.links.getInfo({ url: "https://www.jet.com" })
   .catch(handleError);
 ```
 
+### Transactions
+
+
+##### All
+Transactions are a paged resource.  The response object will contain properties `meta.next` and `meta.previous` which can be supplied to subsequent invocations of `#all` to fetch additional results.
+
+`#all` accepts an optional second parameter, `options` which may define the follow keys to narrow results:
```

Comment:
This reads a bit awkwardly with the colon at the end, then the bit about account ids before the "options" block

Reply:
Will swap the ordering